libs/community/tests/integration_tests/retrievers/document_compressors/test_embeddings_filter.py

```diff
@@ -23,6 +24,21 @@ def test_embeddings_filter() -> None:
     assert len(set(texts[:2]).intersection([d.page_content for d in actual])) == 2
 
 
+@pytest.mark.asyncio
```

Comment:
```suggestion
```

Reply:
if I delete this annotation, the test is not run.